.github/workflows/collection-continuous-integration.yml

```diff
@@ -0,0 +1,258 @@
+        path: .cache/collection-tarballs
+
+  sanity-test-collection-via-vms:
+    name: Sanity in VM ${{ matrix.os.vm || 'ubuntu-latest' }}
+    needs:
+    - build-collection-artifact
+    runs-on: ${{ matrix.os.vm || 'ubuntu-latest' }}
+    strategy:
+      fail-fast: false
+      matrix:
+        ansible-version:
+          - git+https://github.com/ansible/ansible.git@devel
+        os:
+          - vm: ubuntu-16.04
+          - vm: macos-latest
+        python-version:
+          - 3.7
+          - 2.7
+
+    steps:
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Set up pip cache
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ github.ref }}-sanity-VMs
+        restore-keys: |
+          ${{ runner.os }}-pip-
+          ${{ runner.os }}-
+    - name: Install Ansible ${{ matrix.ansible-version }}
+      run: >-
+        python -m pip install --user ${{ matrix.ansible-version }} virtualenv
+    - name: Download migrated collection artifacts
+      uses: actions/download-artifact@v1
+      with:
+        name: >-
+          collection
+        path: .cache/collection-tarballs
+    - name: Install the collection tarball
+      run: >-
+        ~/.local/bin/ansible-galaxy collection install .cache/collection-tarballs/*.tar.gz
+    - name: Run collection sanity tests
+      run: >-
+        ~/.local/bin/ansible-test sanity --color --requirements --venv --python "${{ matrix.python-version }}" -vvv
+      working-directory: >-
+        /${{ runner.os == 'Linux' && 'home' || 'Users' }}/runner/.ansible/collections/ansible_collections/containers/podman
+
+  sanity-test-collection-via-containers:
+    name: Sanity in container via Python ${{ matrix.python-version }}
+    needs:
+    - build-collection-artifact
+    runs-on: ${{ matrix.runner-os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runner-os:
+          - ubuntu-latest
+        runner-python-version:
+          - 3.7
+        ansible-version:
+          - ansible==2.9.5
+          - git+https://github.com/ansible/ansible.git@devel
+        python-version:
+          - 2.7
+          - 3.7
+
+    steps:
+    - name: Set up Python ${{ matrix.runner-python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.runner-python-version }}
+    - name: Set up pip cache
+      uses: actions/cache@v1
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ github.ref }}-sanity-containers
+        restore-keys: |
+          ${{ runner.os }}-pip-
+          ${{ runner.os }}-
+    - name: Install Ansible ${{ matrix.ansible-version }}
+      run: >-
+        python -m pip install --user ${{ matrix.ansible-version }}
+    - name: Download migrated collection artifacts
+      uses: actions/download-artifact@v1
+      with:
+        name: >-
```

Comment:
This field is very short, one word does not need multilining

Reply:
migrate.py does it :)
fixed

Reply:
I know, I authored that for migrate.py. But I only did it in places where I wanted it to fit 80-char line-length (because short lines are better readable. If you look at https://github.com/ansible-community/collection_migration/blob/master/.github/workflows/collection-migration-tests.yml, you'll see that for short names it's not used.
It's also used in places with templating — you cannot split those lines because YAML will render an extra space in the middle.

Another good use case is strings starting with `{` — normally you'd have to quote them but not if `>` or `|` is used.